setup.py

```diff
@@ -32,13 +32,14 @@ def readme():
     ],
     include_package_data = True,
     scripts=['cchecker.py'],
+    # Note: Do not use colons in the entry-point keys. Python 3 reserves
+    # portions of the key after a colon for special use.
```

Comment:
also plz add something like "the strings on the left below *DO NOT MATTER*, the real checker names are read from class attributes"